test/space/ntuples_test.py

```diff
@@ -218,7 +209,7 @@ def test_vector_class_init(fn):
     arr = _array(fn)
 
     FnVector(fn, arr)
-
+"""
```

Comment:
Was removed when I removed the tests in FnVector, but when I re-added that it shoulda been added back in.